hivemind/p2p/p2p_daemon.py

```diff
@@ -382,6 +397,11 @@ def _terminate(self) -> None:
             self._child.wait()
             logger.debug(f'Terminated p2pd with id = {self.id}')
 
+            with suppress(FileNotFoundError):
+                os.remove(self._daemon_listen_maddr['unix'])
+        with suppress(FileNotFoundError):
+            os.remove(self._client_listen_maddr['unix'])
```

Comment:
(not for this PR: maybe we should try contextlib.suppress for other modules where we suppress warnings, e.g. [compression.py](https://github.com/learning-at-home/hivemind/blob/master/hivemind/utils/compression.py#L24))